frontend/package.json

```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "ajv": "^6.7.0",
     "aphrodite": "^2.2.3",
+    "dotenv": "^6.2.0",
```

Comment:
Would this actually be a dev dependency as opposed to a general one?

Reply:
Fair point.